packages/vscode-extension/package.json

```diff
@@ -57,7 +57,6 @@
     "onCommand:workbench.action.debug.start",
     "workspaceContains:/.fx/**/*",
     "workspaceContains:teamsapp.yml",
-    "onView:teamsfx-empty-project",
```

Comment:
This should be replaced by a default view for non-TTK project.

Reply:
Fixed. Thanks.